### PR TITLE
upgrade to polkadot SDK 1.10

### PR DESCRIPTION
**Pull request by brenzi**
* [x] re-run benchmarks
* [x] try-runtime upgrade kusama (checked by CI too)
* [x] try-runtime upgrade paseo (now also checked in CI)

### CODE REVIEW

File: Cargo.lock
Comment (clangenb) [inline Cargo.lock:1]: Lockfile looks ok.

File: polkadot-parachains/src/command.rs [hunk 1]
```diff
@@ -241,157 +242,170 @@ pub fn run() -> Result<()> {
...
-			runner.sync_run(|config| cmd.run(config.chain_spec, config.network))
-		},
-		Some(Subcommand::CheckBlock(cmd)) => {
-			construct_async_run!(|components, cli, cmd, config| {
+        Some(Subcommand::BuildSpec(cmd)) => {
+            let runner = cli.create_runner(cmd)?;
+            runner.sync_run(|config| cmd.run(config.chain_spec, config.network))
+        }
+        Some(Subcommand::CheckBlock(cmd)) => {
+            construct_async_run!(|components, cli, cmd, config| {
```
Comment (clangenb) [inline polkadot-parachains/src/command.rs:250]: This seems weird to me, but in the IDE it looks good. Do you know what happened?

**Review by clangenb — APPROVED** [review]
Looks good to me, I just don't undersand the fmt change. Do you have an explanation for that?

File: polkadot-parachains/src/command.rs (hunk 1 above)
Comment (brenzi) [reply polkadot-parachains/src/command.rs:250]: it disturbed me too. it changed tabs for spaces or vice versa, dunno why. cargo fmt change?

**Pull request merged**